dbms/src/Flash/CoprocessorHandler.cpp

```diff
@@ -162,6 +162,7 @@ grpc::Status CoprocessorHandler<is_stream>::execute()
                 kind,
                 resource_group_name,
                 Logger::get(msg));
+            dag_context.setEnableResourceControl(cop_context.db_context.getSettingsRef().enable_resource_control);
```

Comment:
It seems can be call in `std::optional<QueryExecutorPtr> executeAsPipeline(Context & context, bool internal)`.

Reply:
https://github.com/pingcap/tiflash/blob/2411b2b9b792cea987abbccf7875a45efdc97a46/dbms/src/Flash/executeQuery.cpp#L201